{"name": "not authenticated user -- @1.13 cases", "status": "passed", "steps": [{"name": "Given alien makes get request: /reconciliation/status/", "status": "passed", "start": 1610701820628, "stop": 1610701820653}, {"name": "Then check response of request: Authentication credentials were not provided. is True", "status": "passed", "start": 1610701820654, "stop": 1610701820654}], "parameters": [{"name": "url", "value": "/reconciliation/status/"}, {"name": "method", "value": "get"}, {"name": "bool", "value": "True"}], "start": 1610701820627, "stop": 1610701820654, "uuid": "3230ab34-27f9-4f9c-9d42-a998aa5bad06", "historyId": "d3e18fdcfa3cf024468fb4788d7c344c", "fullName": "features/III-reconciliation_access.feature:not authenticated user", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "reconciliation check access"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}